card/lib/card/format/nesting/subformat.rb

```diff
@@ -0,0 +1,52 @@
+class Card
+  class Format
+    module Nesting
+      # Formats can have subformats. Lineage can be retraced through "parent" format.
+      module Subformat
+        # note: while it is possible to have a subformat of a different class,
+        # the :format_class value takes precedence over :format.
+        def subformat subcard, opts={}
```

Comment:
not sure why it didn't catch on to this renaming (nest to nesting), but the main change here is that now subformat takes opts.